sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/src/Generated/Models/NfsStorageClassTypeProperties.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager.ContainerOrchestratorRuntime.Models
+{
+    /// <summary> The properties of NFS StorageClass. </summary>
+    public partial class NfsStorageClassTypeProperties : StorageClassTypeProperties
+    {
+        /// <summary> Initializes a new instance of <see cref="NfsStorageClassTypeProperties"/>. </summary>
+        /// <param name="server"> NFS Server. </param>
+        /// <param name="share"> NFS share. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="server"/> or <paramref name="share"/> is null. </exception>
+        public NfsStorageClassTypeProperties(string server, string share)
+        {
+            Argument.AssertNotNull(server, nameof(server));
+            Argument.AssertNotNull(share, nameof(share));
+
+            Server = server;
+            Share = share;
+            SCType = SCType.NFS;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="NfsStorageClassTypeProperties"/>. </summary>
+        /// <param name="scType"> Type of the storage class. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        /// <param name="server"> NFS Server. </param>
+        /// <param name="share"> NFS share. </param>
+        /// <param name="subDir"> Sub directory under share. If the sub directory doesn't exist, driver will create it. </param>
+        /// <param name="mountPermissions"> Mounted folder permissions. Default is 0. If set as non-zero, driver will perform `chmod` after mount. </param>
+        /// <param name="onDelete"> The action to take when a NFS volume is deleted. Default is Delete. </param>
+        internal NfsStorageClassTypeProperties(SCType scType, IDictionary<string, BinaryData> serializedAdditionalRawData, string server, string share, string subDir, string mountPermissions, NfsDirectoryActionOnVolumeDeletion? onDelete) : base(scType, serializedAdditionalRawData)
+        {
+            Server = server;
+            Share = share;
+            SubDir = subDir;
+            MountPermissions = mountPermissions;
+            OnDelete = onDelete;
+            SCType = scType;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="NfsStorageClassTypeProperties"/> for deserialization. </summary>
+        internal NfsStorageClassTypeProperties()
+        {
+        }
+
+        /// <summary> NFS Server. </summary>
+        public string Server { get; set; }
```

Comment:
Is this a `Uri`?

Reply:
No. It can be an anything used to access a server, including IP.